tracer/src/Datadog.Trace/Debugger/Instrumentation/LineDebuggerInvoker.cs

```diff
@@ -66,25 +63,12 @@ public static void LogLocal<TLocal>(ref TLocal local, int index, ref LineDebugge
                 return;
             }
 
-            var localNamesFromPdb = state.MethodMetadaInfo.LocalVariableNames;
-            if (localNamesFromPdb != null)
+            var localNamesFromPdb = state.MethodMetadataInfo.LocalVariableNames;
```

Comment:
Seems like this variable should be renamed to just `localVariableNames`? Since it's now also used to hold names of hoisted variables, which technically don't come from the `pdb`?

Reply:
Fixed in 5a103fbe57cedbc9e13568d4372c82a921c46e26